tests/unit/actions/test_default.py

```diff
@@ -95,3 +96,47 @@ def test_default_returns_callable_with_data(
     assert handled
     assert details["responses"][0] == {"key": "k"}
     assert details["responses"][1] == sentinel
+
+
+def test_counter_is_incremented_when_workflows_was_aborted(
+    context_create_example: ActionContext, mocked_bugzilla, mocked_jira
+):
+    context_create_example.jira.project = "FENIX"
+    mocked_bugzilla.get_bug.return_value = context_create_example.bug
+    mocked_jira.create_or_update_issue_remote_links.side_effect = requests.HTTPError(
+        "Unauthorized"
+    )
+    callable_object = default.init(jira_project_key=context_create_example.jira.project)
+
+    with mock.patch("jbi.actions.default.statsd") as mocked:
+        with pytest.raises(requests.HTTPError):
+            callable_object(context=context_create_example)
+
+    mocked.incr.assert_called_with("jbi.action.fenix.aborted.count")
+
+
+def test_counter_is_incremented_when_workflows_was_incomplete(
+    context_create_example: ActionContext, mocked_bugzilla, mocked_jira
+):
+    context_create_example.jira.project = "FENIX"
+    context_create_example.bug.resolution = "WONTFIX"
```

Comment:
This is more a matter of taste, but in these cases where we're overwriting properties of some "example" object, it seems like we could also use a factory.